hotshot-state-prover/src/service.rs

```diff
@@ -200,6 +200,35 @@ pub fn light_client_genesis_from_stake_table(
     Ok(pi.into())
 }
 
+pub async fn light_client_genesis_stake(
+    sequencer_url: &Url,
+    stake_table_capacity: usize,
+) -> anyhow::Result<ParsedStakeState> {
+    let st = init_stake_table_from_sequencer(sequencer_url, stake_table_capacity)
+        .await
+        .with_context(|| "Failed to initialize stake table")?;
+    light_client_genesis_stake_from_stake_table(st)
+}
+
+#[inline]
+pub fn light_client_genesis_stake_from_stake_table(
+    st: StakeTable<BLSPubKey, StateVerKey, CircuitField>,
+) -> anyhow::Result<ParsedStakeState> {
```

Comment:
shouldn't we use `StakeTable.into()` I thought `From<hotshot_stake_table::StakeTable> for ParsedStakeTableState` is implemented in `adapters/light_client.rs`?

if not, we should add it there; and we can remove this function.